homeassistant/components/ecobee/sensor.py

```diff
@@ -46,15 +46,15 @@ class EcobeeSensorEntityDescription(
         native_unit_of_measurement=TEMP_FAHRENHEIT,
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
-        runtime_key="actualTemperature",
+        runtime_key=None,
```

Comment:
Would using Optional[str] in the Mixin class definition resolve this mypy error?

Reply:
Oof I guess pylint didn't like that idea. runtime_key: str | None instead then?